dist/formatters/utils/no-internal-summary-table-creator.js

```diff
@@ -0,0 +1,100 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+const path = require('path');
+const fs = require('fs');
+const process = require('process');
+const createAsciiTable = require('./create-ascii-table');
+
+module.exports = function(messages, ruleId) {
+  const problemFiles = new Map();
+  const errorTracker = new Map();
+  const tagViolationsTracker = new Map();
+  const currentWorkingDirectory = process.cwd();
+  const maxCellWidth = 60;
+  // Ensure the current working directory ends with a path separator
+  const cwdWithSeparator = currentWorkingDirectory.endsWith(path.sep) ? currentWorkingDirectory : currentWorkingDirectory + path.sep;
+
+  messages.forEach((message) => {
+    const errorMessage = message.text ?? message.message;
+    // regex pattern for error message for no internal violations
+    const re = new RegExp('(.*) "(.*)" is (.*).');
+    const match = errorMessage.match(re);
```

Comment:
Was attempting to continue the conversation on this... no idea if it will fall under the past alert.. anyway

Why would it be hard to test the non regex way?

Reply:
it wouldn't be hard to test with the non-regex way but it would be harder to extract the kind, name and tag strings from the message without a regex. testing for the specific rule should be straightforward